config/default.yml

```diff
@@ -34,9 +34,137 @@ Chef/ServiceResource:
   Description: Use a service resource to start and stop services
   Enabled: true
 
-Chef/CopyrightCommentFormat:
+Chef/TmpPath:
+  Description: Use file_cache_path rather than hard-coding tmp paths
+  Enabled: true
+
+RSpec/AnyInstance:
```

Comment:
These rspec configs shouldn't be here. That's leftover from the fork of rubocop-rspec